README.md

```diff
@@ -29,9 +29,11 @@ Configure a repository to use git-crypt:
 
 Specify files to encrypt by creating a .gitattributes file:
 
-    secretfile filter=git-crypt diff=git-crypt
-    *.key filter=git-crypt diff=git-crypt
-    secretdir/** filter=git-crypt diff=git-crypt
+    secretfile eol=lf filter=git-crypt diff=git-crypt
+    *.key eol=lf filter=git-crypt diff=git-crypt
+    secretdir/** eol=lf filter=git-crypt diff=git-crypt
+
+***eol=lf settings is necessary if autocrlf is true. 
```

Comment:
What was the intent of the markup here? I don't think three asterisks is the right way to get anything. Did you mean to just add one escaped asterisk? `\*` should do the trick. If you meant to bold and italic something at the same time you also need a closing somewhere.

Reply:
ah, totally missed that. it is fixed